content/en/docs/release-oversight/troubleshooting-payload-failures.md

```diff
@@ -68,12 +80,19 @@ Some of the highlights contained in the artifacts are
     - pods  - most recent pod logs (but not ephemeral pods)
 - [Gather-network](https://steps.ci.openshift.org/chain/gather-network)
     - Includes network.tar that can be downloaded, untarred and searched / viewed locally
+- [gather-aws-console](https://steps.ci.openshift.org/reference/gather-aws-console), [gather-gcp-console](https://steps.ci.openshift.org/reference/gather-gcp-console), etc.  
```

Comment:
Optional: maybe include the dir for the console/journal logs? e.g., `.../artifacts/e2e-gcp-ovn-upgrade/gather-extra/artifacts/nodes/`)

Reply:
Good feedback.  What I have in there is the links to the steps.  I do have one for gather-extra and mention the kublet logs.  I'm thinking that is ok, otherwise I would have to link to a specific job I think.

Reply:
Ahh, are you just saying add the path like gather-extra/artifacts/nodes?  I could go either way on that.